README.md

```diff
@@ -28,31 +28,159 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## Authentication
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+Planet's APIs require an account for use.
+[Sign up here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+Authentication information can be obtained and stored locally with the CLI
+`planet init` command. Both the Python library and CLI will default to using
+the stored authentication information, if available (not yet implemented, see
+issues #244 and #248).
 
+Additionally, authentication information can be provided to the CLI by
+storing the api key in the environmental variable `PL_API_KEY` or by specifying
+the api key with the flag `-k` or `--api-key` (TODO: link to cli documentation
+here).
 
-## Development
+## Quick Start
 
-To contribute or develop with this library, see
-[CONTRIBUTING](https://github.com/planetlabs/planet-client-python/CONTRIBUTING.md)
+### Python Library Usage
 
+The client modules within the Python library are asynchronous, which greatly
+speeds up many interactions with Planet's APIs. Support for asynchronous
+development is native to Python 3.6+ via the
+[`asyncio` module](https://docs.python.org/3/library/asyncio.html) and is
+fairly easy to implement.
+
+```python
+import asyncio
+import os
+
+import planet
+
+API_KEY = os.getenv('PL_API_KEY')
+
+image_ids = ['3949357_1454705_2020-12-01_241c']
+order_details = planet.OrderDetails(
+    'test_order',
+    [planet.Product(image_ids, 'analytic', 'psorthotile')]
+)
+
+async def create_order(order_details):
+    async with planet.Session(auth=(API_KEY, '')) as ps:
```

Comment:
This appears to be out-of-date based on things like the `planet.get_auth()` conversations we've had recently, and similarly above with the `API_KEY` assignment and `os` import.

Reply:
the authentication work is captured in #248 - this will be changed there.